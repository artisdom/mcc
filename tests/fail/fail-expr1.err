Type error: expected one of [bool] but got int. Error occured in statement:
d + a;