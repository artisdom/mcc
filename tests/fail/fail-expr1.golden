Type error: expected one of [ void *
                            , int
                            , float ] but got bool. Error occured in statement:
d + a;